app/src/main/java/de/westnordost/streetcomplete/quests/place_name/AddPlaceName.kt

```diff
@@ -27,31 +27,67 @@ class AddPlaceName(
+                "social_facility", "nursing_home", "childcare", "retirement_home", "social_centre", // social
+                "ferry_terminal",                                                                   // transport
+                "hospital",                                                                         // health care
+                "place_of_worship", "monastery",                                                    // religious
+                "kindergarten", "school", "college", "university", "research_institute",            // education
+                "driving_school", "dive_centre", "language_school", "music_school",                 // learning
+                "brothel", "gambling", "love_hotel", "stripclub",                                   // bad stuff
+                "animal_boarding", "animal_shelter", "animal_breeding"                              // animals
             ),
             "tourism" to arrayOf(
-                "attraction", "zoo", "aquarium", "theme_park", "gallery", "museum",                                                     // attractions
+                // common
+                "zoo", "aquarium", "theme_park", "gallery", "museum",
+
+                // name only
+                "attraction",                                                                                                           // attractions
                 "hotel", "guest_house", "motel", "hostel", "alpine_hut", "apartment", "resort", "camp_site", "caravan_site", "chalet"   // accommodations
                 // and tourism=information, see above
             ),
             "leisure" to arrayOf(
-                "nature_reserve", "sports_centre", "fitness_centre", "dance", "golf_course",
-                "water_park", "miniature_golf", "stadium", "marina", "bowling_alley",
-                "amusement_arcade", "adult_gaming_centre", "tanning_salon", "horse_riding"
+                //common
+                "fitness_centre", "dance", "golf_course", "water_park",
+                "miniature_golf", "bowling_alley", "horse_riding",  "amusement_arcade",
+                "adult_gaming_centre", "tanning_salon",
+
+                //name only
+                "nature_reserve", "sports_centre", "dance", "golf_course",
+                "stadium", "marina"
+            ),
+            "office" to arrayOf(
+                // common
+                "insurance", "government", "travel_agent", "tax_advisor", "religion", "employment_agency",
+
+                // name and wheelchair
+                "lawyer", "estate_agent", "political_party", "therapist"
+            ),
+            "craft" to arrayOf(
+                // common
+                "carpenter", "shoemaker", "tailor", "photographer", "dressmaker",
+                "electronics_repair", "key_cutter", "stonemason",
+
+                // name and wheelchair
+                "winery"
```

Comment:
The name is already asked for **any** craft, see line 26. This makes no sense here.